pennylane_qiskit/converter.py

```diff
@@ -224,16 +230,43 @@ def _function(params: dict = None, wires: list = None):
                 gate = dagger_map[instruction_name]
                 qml.adjoint(gate)(wires=operation_wires)
 
+            elif isinstance(op, Measure):
+                # Store the current operation wires
+                op_wires = set(operation_wires)
+                # Look-ahead for more gate(s) on its wire(s)
+                meas_terminal = True
+                for next_op, next_qargs, __ in qc.data[idx + 1 :]:
+                    # Check if the subsequent whether next_op is measurement interfering
+                    if not isinstance(next_op, (Barrier, GlobalPhaseGate)):
+                        next_op_wires = set(wire_map[hash(qubit)] for qubit in next_qargs)
+                        # Check if there's any overlapping wires
+                        if next_op_wires.intersection(op_wires):
+                            meas_terminal = False
+                            break
+
+                # Allows for queing the mid-circuit measurements
+                if not meas_terminal:
+                    mid_circ_meas.append(qml.measure(wires=operation_wires))
+                else:
+                    terminal_meas.extend(operation_wires)
+
```

Comment:
```suggestion
                if meas_terminal:
                    terminal_meas.extend(operation_wires)
                else:
                    mid_circ_meas.append(qml.measure(wires=operation_wires))

```

Tends to be a bit more readable to control on the positive, since it's one less thing the brain needs to keep in active memory.